### PR TITLE
docs: add `queue-name` attribute

### PR DESCRIPTION
**Pull request by DouglasBlackwood**
Change-Id: I7e971e93a1eacda2cb077d49eb46346daac65645

### CODE REVIEW

**Comment by DouglasBlackwood** [comment]
This pull request is part of a stack:
1. docs: add `queue-name` attribute ([#3098](https://github.com/Mergifyio/docs/pull/3098))

**Review by AlexandreGaubert — DISMISSED** [review]

File: static/mergify-configuration-openapi.json [hunk 1]
```diff
@@ -779,19 +784,19 @@
...
+        "checks-failed",
+        "queue-rule-missing",
+        "target-branch-missing",
+        "target-branch-changed",
+        "pr-unexpectedly-failed-to-merge",
+        "batch-max-failure-resolution-attempts",
+        "conflict-with-base-branch",
+        "conflict-with-pull-ahead",
+        "branch-update-failed"
+      ]
```
Comment (jd) [inline static/mergify-configuration-openapi.json:799]: unrelated

**Review by jd — CHANGES_REQUESTED** [review]
too much reformating

**Review by jd — APPROVED** [review]

**Review by Kontrolix — APPROVED** [review]
